src/audio/detect_test.c

```diff
@@ -76,6 +77,26 @@ struct comp_data {
 			    struct comp_buffer *source, uint32_t frames);
 };
 
+static inline bool detector_is_sample_width_supported(enum sof_ipc_frame sf)
```

Comment:
Just a note: when function is static/private its name does not have to be unique, so you can make it shorter

Reply:
I would keep it as it is, long but plain, simple and understandable at first sight.